.github/workflows/gradle.yml

```diff
@@ -63,7 +63,27 @@ jobs:
       run: ./gradlew :vividus-tests:runStories -Pvividus.configuration.environments=integration
 
     - name: Publish Integration tests report
+      if: always()
       uses: actions/upload-artifact@v1
       with:
         name: Integration Tests - Allure report
         path: vividus-tests/output/reports/allure
+
+    - name: System tests
+      if: always()
+      env:
+        BROWSERSTACK_USER: ${{ secrets.BROWSERSTACK_USER }}
+        BROWSERSTACK_KEY: ${{ secrets.BROWSERSTACK_KEY }}
+      run: |
+        if [ -n $BROWSERSTACK_USER ] && [ -n $BROWSERSTACK_KEY ]; then
+            ./gradlew :vividus-tests:runStories -Pvividus.configuration.environments=system -Pvividus.configuration.suite=system
+        else
+            echo No BROWSERSTACK_USER or BROWSERSTACK_KEY not available, System tests will be skipped
```

Comment:
`No BROWSERSTACK_USER or BROWSERSTACK_KEY` -> `No BROWSERSTACK_USER and/or BROWSERSTACK_KEY`